### PR TITLE
add configurations for maven and do refactor

### PR DESCRIPTION
**Pull request by olehedza**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: olehedza
+  Date: 25.04.20
+  Time: 16:58
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/index.jsp]: let's remove comments created by Idea

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>mate.academy.webshop.controllers.IndexController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]: always add one empty line to the EOF

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/webapp/web.xml (hunk 2 above)
Comment (olehedza) [reply src/main/webapp/web.xml]: Fixed.

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 1 above)
Comment (olehedza) [reply src/main/webapp/WEB-INF/views/index.jsp]: Fixed.

**Pull request merged**